crypto/BRCryptoWalletManager.c

```diff
@@ -461,6 +445,57 @@ cryptoWalletManagerRemWallet (BRCryptoWalletManager cwm,
     if (NULL != managerWallet) cryptoWalletGive (managerWallet);
 }
 
+static void
+cryptoWalletManagerInstallWalletsForCurrencies (BRCryptoWalletManager cwm) {
+    BRCryptoNetwork  network    = cryptoNetworkTake (cwm->network);
+    BRCryptoCurrency currency   = cryptoNetworkGetCurrency(network);
+    BRCryptoUnit     unitForFee = cryptoNetworkGetUnitAsBase (network, currency);
+
+    size_t currencyCount = cryptoNetworkGetCurrencyCount (network);
+    for (size_t index = 0; index < currencyCount; index++) {
+        BRCryptoCurrency c = cryptoNetworkGetCurrencyAt (network, index);
+        if (c != currency) {
+            BRCryptoUnit unitDefault = cryptoNetworkGetUnitAsDefault (network, c);
+
+            switch (cwm->type) {
+                case BLOCK_CHAIN_TYPE_BTC:
+                    break;
+                case BLOCK_CHAIN_TYPE_ETH: {
+                    const char *address = cryptoCurrencyGetIssuer(c);
+                    if (NULL != address) {
+                        BREthereumGas      ethGasLimit = gasCreate(TOKEN_BRD_DEFAULT_GAS_LIMIT);
+                        BREthereumGasPrice ethGasPrice = gasPriceCreate(etherCreate(createUInt256(TOKEN_BRD_DEFAULT_GAS_PRICE_IN_WEI_UINT64)));
+
+                        tokenInstall (address,
```

Comment:
Naive question; these gas limit / price values, what are the implications of using defaults? Are they updated later on? Does it matter if they are or are not?

Reply:
Our 'estimate fee' interface should supersede these.